src/main/scala/ch/epfl/bluebrain/nexus/kg/search/QueryResultEncoder.scala

```diff
@@ -83,11 +83,17 @@ object QueryResultEncoder {
         }
         graphWithoutListEnd.add(lastElement, rdf.rest, rdf.nil)
     }
+    import ch.epfl.bluebrain.nexus.rdf.syntax.nexus._
     graphWithList
       .asJson(Contexts.searchCtx)
```

Comment:
Whenever you know the `@id` (in your case this localhost trick), you should use it instead. So if you use:
```scala
graphWithList.asJson(searchCtx, url"http://localhost")
```
You can remove `import ch.epfl.bluebrain.nexus.rdf.syntax.nexus._` and put the other import on top without this private method